src/common/post.ts

```diff
@@ -80,3 +83,51 @@ export const fetchLinkPreview = async (
 
   throw new Error('Internal server error!');
 };
+
+export const WELCOME_POST_TITLE =
+  process.env.WELCOME_POST_TITLE ??
+  'Hello World! Start your squad journey here';
+
+const getWelcomeContent = (
+  squadName: string,
```

Comment:
Just to keep it extendable for the future maybe accept `SquadSource`, it can even be `Pick<SquadSource, 'name'>` since if we need any other property function will not change its signature.